modules/nw-networkpolicy-about.adoc

```diff
@@ -56,6 +56,16 @@ spec:
   policyTypes:
   - Ingress
 ----
++
+[TIP]
```

Comment:
I would consider this a warning based on the criteria here: https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

```suggestion
[IMPORTANT]
```